modules/log6x-rn-6.0.2.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+// log6x-release-notes
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-6-0-2_{context}"]
+= Logging 6.0.2
+This release includes link:https://access.redhat.com/errata/RHBA-2024:10051[RHBA-2024:10051].
+
+[id="logging-release-notes-6-0-2-bug-fixes_{context}"]
+== Bug fixes
+
+* Before this update, Loki did not correctly load some configurations, which caused issues when using Alibaba Cloud or IBM Cloud object storage. This update fixes the configuration-loading code in Loki, resolving the issue. (link:https://issues.redhat.com/browse/LOG-5325[LOG-5325])
+
+* Before this update, the collector would discard audit log messages that exceeded the configured threshold. This modifies the audit configuration thresholds for the maximum line size as well as the number of bytes read during a read cycle. (link:https://issues.redhat.com/browse/LOG-5998[LOG-5998])
+
+* Before this update, the operator did not watch and reconcile resources associated with an instance of a ClusterLogForwarder like it did in prior releases. This update modifies the operator to watch and reconcile all resources it owns and creates. (link:https://issues.redhat.com/browse/LOG-6264[LOG-6264])
```

Comment:
which Operator? Either way, even if you don't specify. Also I think that `ClusterLogForwarder` is a CR, right? 

```suggestion
* Before this update, the Operator did not watch and reconcile resources associated with an instance of a `ClusterLogForwarder` CR like it did in prior releases. This update modifies the Operator to watch and reconcile all resources it owns and creates. (link:https://issues.redhat.com/browse/LOG-6264[LOG-6264])
```